article/_posts/2021-06-09-CQL.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: review
+title: "Conservative Q-Learning for Offline Reinforcement Learning"
+tags: reinforcement
+author: ""
+cite:
+    authors: "Aviral Kumar, Aurick Zhou, George Tucker, Sergey Levine"
+    title:   "Conservative Q-Learning for Offline Reinforcement Learning"
+    venue:   "arXiv"
+pdf: "https://arxiv.org/abs/2006.04779"
```

Comment:
I tend to put the link to the PDF as the tag says, but not sure whether that is general rule or a preference.

Reply:
Fixed in 43aa5a7